src/angular-sortable-view.js

```diff
@@ -257,13 +276,19 @@
 
 						if($target){
 							$target.element.removeClass('sv-candidate');
-							var spliced = originatingPart.model(originatingPart.scope).splice(index, 1);
+							var spliced;
+							if(!originatingPart.scope.clone) 
+								spliced = originatingPart.model(originatingPart.scope).splice(index, 1);
+							else { 
+								spliced = [clone(originatingPart.model(originatingPart.scope)[index])];
+								spliced[0].$uniq_key = generateKey();
+							}
```

Comment:
Why are you generating some property in that cumbersome way. Just to have a slightly different object so that the ngRepeat won't scream duplicates? If so wouldn't it be enough to just copy the object?

```javascript
spliced = angular.copy(spliced);
```